### ISSUE TITLE
Forum

### ISSUE THREAD

**Issue by wirrdad** [WikiNarau/wikinarau#42]
Forum for the community with roles/badges

**Comment by Melchizedek6809** [comment]
Might also be nice to add a chat to every page, which could be implemented via a thread that gets created automatically for every entry and only hold the last 100 messages or so.

**Comment by schmueller23** [comment]
hi everyone, i am new to Github, is this the place where post my questions regarding programming?

**Comment by Melchizedek6809** [comment]
@schmueller23 Probably not, StackOverflow might be a better place, this is just an Issue/Ticket for a Project we're working on :)

**Comment by schmueller23** [comment]
@Melchizedek6809 okay then thank you. i'l pop over to stack overflow for help. but is there anything that i can do on git hub to get help or guidancew on my programming or is it jhust for posting projects?

**Comment by Melchizedek6809** [comment]
@schmueller23 GitHub is mainly for hosting projects as well as some services to make programming in a team easier (like this issue tracker). Apart from StackOverflow a lot of frameworks/programming languages also have Discord servers where, in my experience at least, People have been super nice and helpful. Do you have any specific interests like Web Development / Game Development? Maybe I could point you to a specific Forum/Server then :)

**Comment by schmueller23** [comment]
@Melchizedek6809 Yes i am interested in doing web development, however my HTML, CSS  ,and Java skills are still at the beginning, which is why i joined github and discord channels to ask for people's opinions on my programming and may be get a bit of mentorship along the way. i can code my own HTML  page now but it's still not perfect and it looks rough.
